go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.1
 	github.com/pkg/errors v0.9.1
-	github.com/projectcalico/api v0.0.0-20210706154750-8686d6f77130
+	github.com/projectcalico/api v0.0.0-20210713142641-eb76b6b20ac1
```

Comment:
I would expect this to match https://github.com/projectcalico/api/commit/905b702d58de83e1daba9dcaa868322c5fa14511

Reply:
Disregard the previous, I thought this was against master. I do think the pin might need to match https://github.com/projectcalico/api/commit/ce455ccc6b3552781355a4bc448870b7ca4db34c though